app/models/aleph_hint.rb

```diff
@@ -64,10 +64,17 @@ def extract_marc_value(tag, subfield, record)
   # For some reason, we sometimes use the standard 866$u for URLs but other
   # times use 956$u. :facepunch:
   def url(record)
-    if extract_marc_value(856, 'u', record).present?
-      extract_marc_value(856, 'u', record).text
-    else
-      extract_marc_value(956, 'u', record).text
-    end
+    urls = []
+    urls << extract_marc_value(956, 'u', record).map(&:text)
+    urls << extract_marc_value(856, 'u', record).map(&:text)
+    urls.flatten.compact.uniq
+  end
+
+  # Select the first Get URL from the array of urls
+  # NOTE: the datasource was created by selecting records that contained
+  # this url pattern so we make the assumption here that at least one url
```

Comment:
We discussed this and agreed throwing an exception (which is what will happen) was ok. A later commit added more clarification to this note. I'm happy to discuss again if you feel we didn't adequately resolve your concern.